.claude/scripts/surveyor-vocabulary-coverage.test.sh

```
@@ -0,0 +1,279 @@
+# sides, so corpus matching stays exact under it.
+normalize() {
+  sed -E 's/<[A-Za-z][A-Za-z0-9_ .>=-]*>/PLACEHOLDER/g' \
+    | sed -E 's/^[[:space:]]+//; s/[[:space:]]+$//; s/[[:space:]]+/ /g'
+}
+
+# Candidates: inline code spans plus fenced-block lines, kept when they open with
+# a forge verb. Generous on purpose -- the guard does the discriminating.
+extract_commands() {
+  local f=$1
+  {
+    grep -ohE '`[^`]+`' "$f" 2>/dev/null | sed 's/^`//; s/`$//' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)